schema/Retail/Update/on_update.ts

```diff
@@ -75,7 +75,7 @@ export const onUpdateSchema = {
           type: 'object',
           properties: {
             id: { type: 'string',
-            pattern: '^[a-zA-Z0-9]{1,32}$',
+            pattern: '^[a-zA-Z0-9-]{1,32}$|^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$',
             errorMessage: 'Should be alphanumeric upto 32 letters max',
            },
             state: { type: 'string', enum: ['Created', 'Accepted', 'Cancelled', 'Completed', 'Delivered'] },
```

Comment:
In the case of return flow (fulfillment type='Return'), the start location ID will not be present. Instead, the start and end locations are swapped, and the location ID of the buyer does not hold any significance

Reply:
will address this in next MR ASAP,
can we merge this for now so other merge conflicts dont arise